src/main/java/com/google/sps/data/User.java

```diff
@@ -65,8 +65,8 @@ public void addSpecialty(String toAdd) {
   }
 
   public void setProfileType(ProfileType input) {
-      if(input == ProfileType.STUDENT) {
-        this.userType = ProfileType.STUDENT;
+      if(input == ProfileType.MENTEE) {
```

Comment:
nit: Slightly more succinct as
`if (input == ProfileType.MENTOR || input == ProfileType.MENTEE) {
  this.userType = input;
} else { ...`

Reply:
Done